packages/react/src/vm-extension/mock-vm-ext-items.ts

```diff
@@ -0,0 +1,92 @@
+import { VmExtItem } from "./vm-extension-list";
+
+export const succeededExtItem: VmExtItem = {
```

Comment:
Can we move this file to the __tests__ folder if this isn't referenced outside of the local tests?

Reply:
This is used in playground as well to display VM extension list demo page, so distributing it as well.

Reply:
Ah, I missed that. Can we move this to a folder which makes it clear this is for tests? Could be either the service package's test-util folder, or one in the react package. Just want to be careful mixing test data and code with production code (even though I know there's an overlap when it comes to the playground).

Reply:
moved to @batch/ui-react package's test-util